sbws/lib/relaylist.py

```diff
@@ -415,28 +422,49 @@ def _init_relays(self):
         # Change to stem.descriptor.remote in future refactor.
         network_statuses = c.get_network_statuses()
         new_relays_dict = dict([(r.fingerprint, r) for r in network_statuses])
+        log.debug("Number of relays in the current consensus: %s.",
+                  len(new_relays_dict))
```

Comment:
`%d` instead of `%s`?

Reply:
hmm, it's python, i think it doesn't really matter. I changed, then changed the rest of logging lines where i'm using a number